### PR TITLE
Comment Activity Log

### PR DESCRIPTION
**Pull request by Rownaknira**

### CODE REVIEW

**Comment by abdullahallmehedi** [comment]
ok

**Comment by SSalekin** [comment]
ok

**Comment by noor-framgia** [comment]
ok

File: app/models/comment.rb [hunk 1]
```diff
@@ -3,4 +3,11 @@ class Comment < ActiveRecord::Base
   tracked owner: ->(controller, model) {controller && controller.current_user}
   belongs_to :user
   belongs_to :review
+
+  class << self
+    def comment_acitvities
```
Comment (dieunb) [inline app/models/comment.rb]: `activities`

**Pull request merged**